tests/api/crud/test_files.py

```diff
@@ -22,41 +22,58 @@
 import server.api.crud
 
 
+@pytest.mark.parametrize(
+    "auth_info, user_secrets, expected_secrets",
```

Comment:
The auth_info parameter is the same for both tests, so you can include it in the test itself rather than in the parametrize decorator.